src/index.html

```diff
@@ -1,12 +1,306 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+  <header class="header">
+    <div class="header__container">
+      <div class="header__logo">
+        <a href="#home"><img src="images/logo.png" alt="logo-moyo"></a>
+      </div>
+      <nav>
+        <ul class="nav-list">
+          <li class="nav-list__li"><a class="nav-list__link is-active" href="#">Apple</a></li>
```

Comment:
```suggestion
          <li class="nav-list__li">
            <a class="nav-list__link is-active" href="#">
              Apple
            </a>
          </li>
```

Не совсем ошибка, но когда все написано в одну строку, читать неудобно